software/nosql/src/main/java/brooklyn/entity/nosql/riak/RiakNodeImpl.java

```diff
@@ -161,8 +162,15 @@ public void connectSensors() {
                                     }
                                 }
                         ))
-                        .onFailureOrException(Functions.constant(Arrays.asList(new String[0]))))
-                .build();
+                        .onFailureOrException(Functions.constant(Arrays.asList(new String[0]))));
+
+        for(AttributeSensor<String> sensor : ONE_MINUTE_STRING_SENSORS) {
+            httpFeedBuilder.poll(new HttpPollConfig<String>(sensor)
+                    .onSuccess(HttpValueFunctions.jsonContents(sensor.getName(), String.class))
+                    .onFailureOrException(Functions.constant("-1")));
```

Comment:
Why are we returning a string with a number in it? If these are numeric sensors, then parse the results with `Integer#valueOf(String)` or similar, so they can be used in policies.